index.html

```diff
@@ -3225,36 +3235,37 @@ <h3>Terms of Use</h3>
     "https://www.w3.org/ns/credentials/v2",
     "https://www.w3.org/ns/credentials/examples/v2"
   ],
-  "id": "http://university.example/credentials/3732",
-  "type": ["VerifiableCredential", "ExampleDegreeCredential"],
-  "issuer": "https://university.example/issuers/14",
-  "validFrom": "2010-01-01T19:23:24Z",
-  "credentialSubject": {
-    "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
-    "degree": {
-      "type": "ExampleBachelorDegree",
-      "name": "Bachelor of Science and Arts"
-    }
+  id: "urn:did:123456",
+  type: ["VerifiableCredential", "VerifiableAttestation", "VerifiableId"],
+  issuer: "did:ebsi:zz7XsC9ixAXuZecoD9sZEM1",
+  issuanceDate: "2021-11-01T00:00:00Z",
+  validFrom: "2021-11-01T00:00:00Z",
+  issued: "2021-10-30T00:00:00Z",
+  credentialSubject: {
+    id: "did:key:z2dmzD81cgPx8Vki7JbuuMmFYrWPgYoytykUZ3eyqht1j9KbrDt4zxXoDrBWYFiATYZ8G9JMeEXC7Kki24fbTwtsJbGe5qcbkYFunSzcDokMRmj8UJ1PbdCGh33mf97K3To89bMzd15qrYq3VkDztoZqfmujkJVpvTbqoXWXqxmzNDbvMJ",
```

Comment:
I don't recommend this "did key" format, I suspect its the JWK format... It might be better to just generalize this to "did:example:123"...

Reply:
I am using the value used by EBSI in its documentation. If I change it, the example will no longer work because the JWK referred to contains this key. I cannot change the JWK because I do not have the signing key.